PWGHF/Core/DecayChannels.h

```diff
@@ -177,4 +311,108 @@ enum DecayChannelResonant : int8_t {
 } // namespace hf_cand_beauty
 } // namespace o2::hf_decay
 
+namespace o2::hf_corrbkg
+{
+using namespace o2::hf_decay;
+
+/// Returns a map of the possible final states for a specific 3-prong particle specie
+/// \param pdgMother PDG code of the mother particle
+/// \return a map of final states with their corresponding PDG codes
+std::unordered_map<hf_cand_3prong::DecayChannelMain, std::vector<int>> getDecayChannel3Prong(int pdgMother)
+{
+  switch (pdgMother) {
```

Comment:
These utility functions should go into a separate utility header, e.g. `Utils/utilsMcMatching.h` together with the definitions of the maps which add constructors in every include. Plus, the matching utilities are only needed in the candidate creators and the McGen utils. They are not needed in any other workflows which only need to compare the flag columns with the enums.